groth16/src/prover.rs

```diff
@@ -240,7 +240,7 @@ where
 
     let witness_map_time = start_timer!(|| "R1CS to QAP witness map");
     let (full_input_assignment, h, _) =
-        R1CStoQAP::witness_map::<E>(&prover, &E::Fr::zero(), &E::Fr::zero(), &E::Fr::zero())?;
+        R1CStoQAP::witness_map::<E>(&prover)?;
```

Comment:
fmt is failing because this should now be on the previous line =/

Reply:
Oops...